components/python/pyzmq/Makefile

```diff
@@ -10,15 +10,32 @@
 
 #
 # Copyright (c) 2014 Alexander Pyhalov.  All rights reserved.
+# Copyright 2016 Jim Klimov
 #
 
+# See zeromq configure.ac for ABI version mapping to software version
```

Comment:
Please, move it below metadata definition, somewhere between "include $(WS_TOP)/make-rules/ips.mk" and  "# common targets".